src/ApiPort/ApiPort.Offline.csproj

```diff
@@ -0,0 +1,65 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <AssemblyName>ApiPort</AssemblyName>
+    <RootNamespace>ApiPort</RootNamespace>
+    <TargetFrameworks>net46;netcoreapp1.0</TargetFrameworks>
+    <RuntimeIdentifiers>win7-x64;win7-x86;osx.10.10-x64;ubuntu.14.04-x64</RuntimeIdentifiers>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <GenerateBindingRedirectsOutputType>true</GenerateBindingRedirectsOutputType>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net46'">
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
+    <DefineConstants>$(DefineConstants);FEATURE_SYSTEM_PROXY;FEATURE_NETWORK_CREDENTIAL</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <!-- Projects default to being signed, but Microsoft.Configuration.* assemblies are not signed -->
```

Comment:
Maybe in a unrelated PR.. but Microsoft.Extensions.Configuration is signed now... Should we remove this?

Reply:
Yeah - but we should probably switch to Microsoft.CommandLine from https://github.com/dotnet/corefxlab